templates/components/itilobject/timeline/timeline_item_header_badges.html.twig

```diff
@@ -57,11 +63,18 @@
 </span>
 
 {% if users_id_editor > 0 and date_creation != date_mod %}
-   <span class="badge text-wrap ms-1 d-none d-md-block">
+   <span class="badge user-select-auto text-wrap ms-1 d-none d-md-block">
       {% set date_span %}
          <span title="{{ date_mod|formatted_datetime }}"
```

Comment:
The tooltip is only relevant when relative dates are displayed.

Reply:
done